### ISSUE TITLE
simnet模型库训练过程 loss全为0

### ISSUE THREAD

**Issue by sshilei** [PaddlePaddle/Paddle#16170]
为使您的问题得到快速解决，在建立Issue前，请您先通过如下方式搜索是否有相似问题:【搜索issue关键字】【使用labels筛选】【官方文档】

建立issue时，为快速解决问题，请您根据使用情况给出如下信息：
- 标题：简洁、精准描述您的问题，例如“ssd 模型前置lstm报错  ”
- 版本、环境信息：
    1）PaddlePaddle版本：fluid 1.3

- 模型信息
    1）simnet

simnet采用pointwise.json 配置时，loss不为0，正常。 使用pairwise.json时，loss全为0.
不管是demo数据，还是真实训练数据，都是如此。

http://10.255.120.13:8388/?path=/home/matrix/containers/372023034.job-0bb5c87810c1005f-trainer/upper/home/crim/

~/paddlejob/workspace/env_run ~/paddlejob/workspace
start cmd is: python paddle_simnet.py --task_type train --conf_file_path examples/cnn_pairwise.json
{u'loss': {u'class_name': u'HingeLoss', u'module_name': u'hinge_loss', u'margin': 0.1}, u'optimizer': {u'class_name': u'AdamOptimizer', u'epsilon': 1e-08, u'learning_rate': 0.001, u'beta2': 0.999, u'beta1': 0.9}, u'dict_size': 200002, u'num_threads': 4, u'test_file_path': u'./test_data/', u'batch_size': 1024, u'use_cuda': 1, u'use_epoch': 0, u'train_file_path': u'./train_data/', u'result_file_path': u'result_cnn_pairwise', u'net': {u'class_name': u'CNN', u'emb_dim': 128, u'filter_size': 3, u'hidden_dim': 128, u'num_filters': 256, u'module_name': u'cnn'}, u'epoch_num': 20, u'task_mode': u'pairwise', u'model_path': u'models/cnn_pairwise'}
W0312 11:41:04.959919 76681 device_context.cc:263] Please NOTE: device: 0, CUDA Capability: 61, Driver API Version: 9.0, Runtime API Version: 9.0
W0312 11:41:04.960033 76681 device_context.cc:271] device: 0, cuDNN Version: 7.0.
2019-03-12 11:41:05,008 - INFO - device count: 1
2019-03-12 11:41:05,008 - INFO - start train process ...
epoch: 0, iter: 99, loss: 0.001100
epoch: 0, iter: 199, loss: 0.000000
epoch: 0, iter: 299, loss: 0.000000
epoch: 0, iter: 399, loss: 0.000000
epoch: 0, iter: 499, loss: 0.000000
epoch: 0, iter: 599, loss: 0.000000
epoch: 0, iter: 699, loss: 0.000000
epoch: 0, iter: 799, loss: 0.000000
epoch: 0, iter: 899, loss: 0.000000
epoch: 0, iter: 999, loss: 0.000000
epoch: 0, iter: 1099, loss: 0.000000
epoch: 0, iter: 1199, loss: 0.000000
epoch: 0, iter: 1299, loss: 0.000000
epoch: 0, iter: 1399, loss: 0.000000
epoch: 0, iter: 1499, loss: 0.000000
epoch: 0, iter: 1599, loss: 0.000000
epoch: 0, iter: 1699, loss: 0.000000
epoch: 0, iter: 1799, loss: 0.000000
epoch: 0, iter: 1899, loss: 0.000000
epoch: 0, iter: 1999, loss: 0.000000
epoch: 0, iter: 2099, loss: 0.000000
epoch: 0, iter: 2199, loss: 0.000000
epoch: 0, iter: 2299, loss: 0.000000
epoch: 0, iter: 2399, loss: 0.000000
epoch: 0, iter: 2499, loss: 0.000000
epoch: 0, iter: 2599, loss: 0.000000
epoch: 0, iter: 2699, loss: 0.000000
epoch: 0, iter: 2799, loss: 0.000000
epoch: 0, iter: 2899, loss: 0.000000
epoch: 0, iter: 2999, loss: 0.000000
epoch: 0, iter: 3099, loss: 0.000000
epoch: 0, iter: 3199, loss: 0.000000
epoch: 0, iter: 3299, loss: 0.000000
epoch: 0, iter: 3399, loss: 0.000000
epoch: 0, iter: 3499, loss: 0.000000
epoch: 0, iter: 3599, loss: 0.000000
epoch: 0, iter: 3699, loss: 0.000000
epoch: 0, iter: 3799, loss: 0.000000

**Comment by AIpioneer** [comment]
This issue is closed because it is fixed.
